README.md

```diff
@@ -137,19 +137,22 @@ Authentication in the default configuration is done through the RedHat SSO, so y
 To authenticate, use the ocm tool against your local service. The ocm tool is available on https://console.redhat.com/openshift/downloads
 
 #### Login to your local service
-```
+
+```shell
+
 ocm login --token=${OCM_ACCESS_TOKEN} --url=http://localhost:8000
 
 ```
 
-#### Get a new Resource
-This will be empty if no Resource is ever created
+#### List the resource bundle
```

Comment:
change it to `List the consumer`. in this step, the result is empty. then create a consumer and list again.

Reply:
done